_sub/compute/helm-atlantis/values/values.yaml

```diff
@@ -4,9 +4,11 @@ image:
   pullPolicy: IfNotPresent
 
 resources:
+  requests:
+    memory: 100Mi
+    cpu: 50m
   limits:
-    memory: 2Gi
-    cpu: 1
+    memory: 100Mi
```

Comment:
why there is no limit on the CPU usage?

Reply:
I was following some advice that specifying CPU limits is actually a bad idea in Kubernetes: https://home.robusta.dev/blog/stop-using-cpu-limits